Caddyfile

```diff
@@ -11,9 +11,16 @@ https://gpuctl.perial.co.uk {
 
 		X-Clacks-Overhead "GNU Terry Pratchett"
 
-		# TODO: allow the API to be accessed from other endpoints
-		# This will need CSRF protection
+		# When serving under the webapi, we only want API requests
+		# from our own origin. So remove the CORS headers that are
+		# to allow cross-origin for local-dev
+
+		# If we want to allow cross-origin api requests, we need
+		# to think carefully about CSRF, but we don't need that for
+		# non-browser stuff, so it's fine.
 		-Access-Control-Allow-Origin
+		-Access-Control-Allow-Methods
+		-Access-Control-Allow-Headers
```

Comment:
I don't think we want to remove these headers if we want this to work in production. I remove the `Access-Control-Allow-Origin` header because in dev, we allow for requests to be made from `localhost:3000`, which never happens in prod. However, if I've read MDN right, `-Methods` and `-Headers` apply to all requests, not just ones from different origins

Reply:
Oooh, where does it say that?

Reply:
It doesn't explicitly say that, but compared to the page for https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Allow-Origin, the pages for https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Request-Headers and https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Allow-Methods make no reference to origin at all, implying they apply to all requests